org-code-javabuilder/protocol/src/main/java/org/code/protocol/GlobalProtocol.java

```diff
@@ -0,0 +1,32 @@
+package org.code.protocol;
+
+public class GlobalProtocol {
```

Comment:
nit: add a comment on this class explaining its purpose

Reply:
✔️